testing/libfixposix/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+pkgname=libfixposix
+pkgver=0.4.3
+pkgrel=1.0
+pkgdesc="Thin wrapper over POSIX syscalls"
+url="https://github.com/sionescu/libfixposix"
+arch="x86_64"
+license="BSL-1.0"
+makedepends="autoconf automake libtool"
+source="$pkgname-$pkgver.tar.gz::https://github.com/sionescu/libfixposix/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
```

Comment:
can be removed, it is the default value

Reply:
The makedepends right? Making sure I'm reading that one correctly